pkg/inject/envoy.go

```diff
@@ -61,6 +69,22 @@ const envoyContainerTemplate = `
     {
       "name": "ENABLE_ENVOY_XRAY_TRACING",
       "value": "1"
+    }{{ end }}{{ if .EnableXrayTracing }},
+    {
+      "name": "XRAY_DAEMON_PORT",
+      "value": "{{ .XrayDaemonPort }}"
+    }{{ end }}{{ if .EnableDatadogTracing }},
+    {
+      "name": "ENABLE_ENVOY_DATADOG_TRACING",
```

Comment:
Is this supposed to replace the static file based config we do here? [1]

[1] https://github.com/aws/aws-app-mesh-controller-for-k8s/blob/master/pkg/inject/datadog.go